misk/src/main/kotlin/misk/web/DashboardTab.kt

```diff
@@ -0,0 +1,54 @@
+package misk.web
+
+import misk.security.authz.AccessAnnotationEntry
+import javax.inject.Inject
+import javax.inject.Provider
+import kotlin.reflect.KClass
+
+class DashboardTab(
+  slug: String,
+  url_path_prefix: String,
+  val name: String,
+  val category: String = "Admin",
+  capabilities: Set<String> = setOf(),
```

Comment:
do we actually want defaults for this?

Reply:
I think so, though I'd be open to removing the default and forcing developers to deliberately choose authentication vs. being open by default. Is that what you'd recommend?

Reply:
> deliberately choose authentication vs. being open by default

I think being open by default seems like a dangerous default.